templates/50_singularity.config.erb

```diff
@@ -5,10 +5,11 @@ SINGULARITY = <%= @singularity_path %>
 SINGULARITY_JOB = <%= @force_singularity_jobs %>
 
 # Forces all jobs to use the given image.
-SINGULARITY_IMAGE_EXPR = "<%= @singularity_image_expr %>"
+SINGULARITY_IMAGE_EXPR = <%= @singularity_image_expr %>
```

Comment:
This should be in quotes, shouldn't it?
I just put a fix for this in version 2.0.7.

Are you configuring your installation as per (now fixed) example, or via hiera?

Reply:
Putting quotes here would be very wrong, see the example in #91 . 
It's an expression, so users should be allowed to put variables there, or something like
`ifThenElse()`. 
Enforcing double quotes in the template removes functionality since it means users can only put a single string there, i.e. one single image. If users want that, they have to specify double quotes in the puppet configuration.

Reply:
PS: #91 also explains how we actually use it ;-). 
In our case, it's even more complex, since we have an external script generating this part of the HTCondor configuration for us.